File: java/Dedicnost/src/main/java/org/example/Programator.java
package org.example;

public class Programator extends Clovek {

    private String jazyk;

    public Programator(String jmeno, int vek, String jazyk) {

        super(jmeno, vek);
        this.jazyk = jazyk;
    }

    @Override
    public void Pozdrav()
    {
        System.out.println("Ahoj posila " + jmeno + " ve veku " +  vek + " je dospely " + jeDospely + " ze tridy Programator programuje v " + jazyk);
    }

    @Override
    public String toString()
    {
        return super.toString() + " " + jmeno + " " + vek + " " + jazyk;
    }

    public String getJazyk() {
        return jazyk;
    }

    public void setJazyk(String jazyk) {
        this.jazyk = jazyk;
    }
}

File: csharp/Dedicnost/Dedicnost/Programator.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Dedicnost
{
    internal class Programator: Clovek
    {
        public string Jazyk { get; set; }

        public Programator(string jmeno, int vek, string jazyk): base(jmeno, vek) {

            Jazyk = jazyk;        
        }

        public override void Pozdrav()
        {
            Console.WriteLine($"Ahoj posila {Jmeno} ve veku {Vek} je dospely {jeDospely} ze tridy Programator programuje v {Jazyk}");
        }

        public override string ToString()
        {
            return base.ToString() + $"{Jmeno} + {Vek} + {Jazyk}";
        }
    }
}
